client/checkout/woopay/express-button/express-checkout-iframe.js

```diff
@@ -250,7 +254,9 @@ export const expressCheckoutIframe = async ( api, context, emailSelector ) => {
 					true
 				);
 				if ( e.data.redirectUrl ) {
-					window.location = e.data.redirectUrl;
+					window.location = appendRedirectionParams(
```

Comment:
This flow does not call the `api.initWooPay` method. So the `order_id` is not passed into the session creation flow. Is that intentional?

Reply:
Do you mean we should remove `appendRedirectionParams` here?

Reply:
Fixed here [6d0cd3c](https://github.com/Automattic/woocommerce-payments/pull/7112/commits/6d0cd3c39dfbf681b7af6e06a07a36ac23205293)